hunting/linux/queries/privilege_escalation_via_process_capabilities.toml

```diff
@@ -0,0 +1,48 @@
+[hunt]
+author = "Elastic"
+description = """
+This hunt identifies processes on Linux systems with specific capabilities set. It monitors process execution events where processes have effective or permitted capabilities, which can be indicative of elevated privileges. The hunt focuses on non-root users to detect potential privilege escalation attempts. The hunt lists detailed information for further analysis and investigation.
+"""
+integration = ["endpoint"]
+uuid = "6f67704d-e5b1-4613-912c-e2965660fe17"
+name = "Process Capability Hunting"
+language = ["ES|QL"]
+license = "Elastic License v2"
+notes = [
+    "Monitors process execution events where processes have specific capabilities set, such as CAP_SYS_MODULE, CAP_SYS_PTRACE, and others.",
+    "Excludes certain processes and capabilities to reduce false positives, but these can be adjusted based on your environment.",
+    "Uses EVAL to tag potential privilege escalation events and counts occurrences to identify unusual activity.",
+    "Focuses on non-root users to detect potential privilege escalation attempts.",
+    "Requires additional data analysis and investigation into results to identify malicious or unauthorized use of process capabilities."
+]
+mitre = ["T1548.001", "T1548.003"]
+
+query = [
+'''
+from logs-endpoint.events.process-*
+| where @timestamp > now() - 30 day
+| where host.os.type == "linux" and event.action == "exec" and event.type == "start" and (process.thread.capabilities.effective is not null or process.thread.capabilities.permitted is not null) and user.id != "0" and
+not (
+  // Remove these if you expect persistence through capabilities
+  process.executable like "/var/lib/docker/*" or
+  process.name == "gnome-keyring-daemon" or
+  process.thread.capabilities.permitted == "CAP_WAKE_ALARM"
+)
+| stats cc = count(), host_count = count_distinct(host.name) by process.parent.executable, process.executable, process.command_line, process.thread.capabilities.effective, process.thread.capabilities.permitted, user.id
```

Comment:
can't we limit this to by process.executbale and capabilities (too many aggregation pts reduce perf)